addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -397,6 +403,7 @@ def _do_action_change_period(self, move_vals):
         accrual_move_offsets = defaultdict(int)
         for move in self.move_line_ids.move_id:
             amount = sum((self.move_line_ids._origin & move.line_ids).mapped('balance'))
+            # TODO: check: `m` vs `move`
```

Comment:
:eyes:

Reply:
Removed. Discussed with andu (who wrote the line) and the way it is should lead to right result (I just did it wrong in a previous version). We try to compute the date of `m` and not `move`.